src/hotspot/share/gc/shared/genArguments.cpp

```diff
@@ -301,8 +301,6 @@ void GenArguments::initialize_size_info() {
     // generation to fit as well.
     // If the user has explicitly set an OldSize that is inconsistent
     // with other command line flags, issue a warning.
-    // The generation minimums and the overall heap minimum should
-    // be within one generation alignment.
     if (initial_old_size > MaxOldSize) {
       log_warning(gc, ergo)("Inconsistency between maximum heap size and maximum "
```

Comment:
JDK-8331432 also mentions this warning message as needing cleanup.
It should be similar to the one below about initial vs min old size, rather than incorrectly mentioning
max heap size and printing MaxHeapSize.
Except it would be useful if both includes both the initial and bound values.
Also, prefer "%zu" rather than SIZE_FORMAT when updating the messages.